CONTRIBUTING.md

```diff
@@ -39,24 +39,16 @@ If your contribution is significantly big, it is better to first check with the
 
 ## Project orchestration
 
-This project uses [Rush](https://rushjs.io) to manage many of our Azure SDK libraries within a single repository. It is highly recommended that you read the [Rush Developer Tutorials](https://rushjs.io/pages/developer/new_developer/) to familiarize yourself with the tool.
+This project uses [pnpm](https://pnpm.io/) to manage many of our Azure SDK libraries within a single repository. It is highly recommended that you read the [pnpm Documentation](https://pnpm.io/motivation) to familiarize yourself with the tool.
 
-Rush provides many benefits:
+PNPM provides many benefits:
```

Comment:
just curious of what our main motivation migrating from rush to pnpm is, i notice we didn't mention that in original issue.

Reply:
The motivation is as follows:
- The development pace of Rush has slowed, no longer evolving as fast as we would want
- We want to take full advantage of the underlying pnpm that we are already using, but Rush has its own mechanisms for doing so, like patches, workspaces, and so forth
- We want to take avantage of a number of pnpm features that Rush would not allow us to do such as workspaces with catalogs and named catalogs
- We want to use a better build and build cache system with TurboRepo which allows us to build more in parallel.